.github/workflows/gradle.yml

```diff
@@ -1,34 +1,50 @@
-# This workflow uses actions that are not certified by GitHub.
-# They are provided by a third-party and are governed by
-# separate terms of service, privacy policy, and support
-# documentation.
-# This workflow will build a Java project with Gradle and cache/restore any dependencies to improve the workflow execution time
-# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-java-with-gradle
-
-name: Java CI with Gradle
+name: Update SDKs
 
 on:
-  push:
-    branches: [ "main" ]
+  workflow_dispatch:
   pull_request:
```

Comment:
The idea here is that we push generated specs as we update versions/specfiles etc?

Reply:
On `pull_request` for us to test changes and `workflow_dispatch` to handle notifications from https://github.com/Adyen/adyen-openapi/blob/main/.github/workflows/notify.yml.

Reply:
Yeah ofc the automation should run, but was thinking we could also automatically generate new models/services if we update version number or anyhing else on the specs